RELEASE.md

```diff
@@ -1,3 +1,29 @@
+# Release 0.1.0
+
+## New features
+- Telemetry. Marqo now includes various timing metrics for many API endpoints when 
+the query parameter `telemetry=True` is specified (https://github.com/marqo-ai/marqo/pull/506).
+- Consolidate default device to CUDA when available (https://github.com/marqo-ai/marqo/pull/508).
```

Comment:
I would add a small blurb about the benefits of new features to users. Also add a link to the docs for further reading/usage.

"
By default, Marqo now uses CUDA devices for search and indexing if available. See [here](https://docs.marqo.ai/0.1.0/API-Reference/search/#query-parameters) for more information. This helps ensure you get the best indexing and search experience without having to explicitly add the `device` parameter to search and add_documents calls. 
"